src/devices/src/virtio/block/device.rs

```diff
@@ -267,14 +289,10 @@ impl Block {
     }
 
     /// Update the backing file for the Block device.
```

Comment:
Update the comment: `Update the backing file and the config space of the block device`.